src/test/javascript/portal/ui/openlayers/control/SpatialConstraintSpec.js

```diff
@@ -116,16 +116,6 @@ describe('Portal.ui.openlayers.control.SpatialConstraint', function() {
             expect(eventSpy).not.toHaveBeenCalled();
         });
 
-        it('does not clear constraint where viewport area is below minimum', function() {
```

Comment:
the test was removed as it 'does' clear constraint where viewport area is below minimum